doc/apiref.rst

```diff
@@ -497,6 +497,11 @@ A JSON array is an ordered collection of other JSON values.
    if *array* is *NULL*, or if *index* is out of range, *NULL* is
    returned.
 
+.. function:: json_t *json_array_contains(const json_t *array, json_t *value)
+
+   Returns 1 if *array* contains *value* and 0 otherwise. That it uses
+   `json_equal()` to compare two values.
+
```

Comment:
Please add `.. versionadded:: 2.10` here, because 2.10 will be the first version that has this function.